README.md

```diff
@@ -231,25 +239,25 @@ git cf
 These are passed to the according `git` calls.
 
-| Command  | Option                                                                      |
-| -------- | --------------------------------------------------------------------------- |
-| `ga`     | `FORGIT_ADD_GIT_OPTS`                                                       |
-| `glo`    | `FORGIT_LOG_GIT_OPTS`                                                       |
-| `gd`     | `FORGIT_DIFF_GIT_OPTS`                                                      |
-| `grh`    | `FORGIT_RESET_HEAD_GIT_OPTS`                                                |
-| `gcf`    | `FORGIT_CHECKOUT_FILE_GIT_OPTS`                                             |
-| `gcb`    | `FORGIT_CHECKOUT_BRANCH_GIT_OPTS`, `FORGIT_CHECKOUT_BRANCH_BRANCH_GIT_OPTS` |
-| `gbd`    | `FORGIT_BRANCH_DELETE_GIT_OPTS`                                             |
-| `gct`    | `FORGIT_CHECKOUT_TAG_GIT_OPTS`                                              |
-| `gco`    | `FORGIT_CHECKOUT_COMMIT_GIT_OPTS`                                           |
-| `grc`    | `FORGIT_REVERT_COMMIT_GIT_OPTS`                                             |
-| `gss`    | `FORGIT_STASH_SHOW_GIT_OPTS`                                                |
-| `gsp`    | `FORGIT_STASH_PUSH_GIT_OPTS`                                                |
-| `gclean` | `FORGIT_CLEAN_GIT_OPTS`                                                     |
-| `grb`    | `FORGIT_REBASE_GIT_OPTS`                                                    |
-| `gbl`    | `FORGIT_BLAME_GIT_OPTS`                                                     |
-| `gfu`    | `FORGIT_FIXUP_GIT_OPTS`                                                     |
-| `gcp`    | `FORGIT_CHERRY_PICK_GIT_OPTS`                                               |
+| Command  | Option                                                                       |
+|----------|------------------------------------------------------------------------------|
+| `ga`     | `FORGIT_ADD_GIT_OPTS`, `FORGIT_ADD_PREVIEW_GIT_OPTS`                         |
+| `glo`    | `FORGIT_LOG_GIT_OPTS`, `FORGIT_LOG_PREVIEW_GIT_OPTS`                         |
+| `gd`     | `FORGIT_DIFF_GIT_OPTS`                                                       |
+| `grh`    | `FORGIT_RESET_HEAD_GIT_OPTS`, `FORGIT_RESET_HEAD_PREVIEW_GIT_OPTS`           |
+| `gcf`    | `FORGIT_CHECKOUT_FILE_GIT_OPTS`, `FORGIT_CHECKOUT_FILE_PREVIEW_GIT_OPTS`     |
+| `gcb`    | `FORGIT_CHECKOUT_BRANCH_GIT_OPTS`, `FORGIT_CHECKOUT_BRANCH_BRANCH_GIT_OPTS`  |
+| `gbd`    | `FORGIT_BRANCH_DELETE_GIT_OPTS`                                              |
+| `gct`    | `FORGIT_CHECKOUT_TAG_GIT_OPTS`                                               |
+| `gco`    | `FORGIT_CHECKOUT_COMMIT_GIT_OPTS`, `FORGIT_CHECKOUT_COMMIT_PREVIEW_GIT_OPTS` |
+| `grc`    | `FORGIT_REVERT_COMMIT_GIT_OPTS`, `FORGIT_REVERT_PREVIEW_GIT_OPTS`            |
```

Comment:
Wondering whether we should go with `FORGIT_REVERT_COMMIT_PREVIEW_GIT_OPTS` here. The name is a bit long, but it would be consistent with the existing `FORGIT_REVERT_COMMIT_GIT_OPTS`.